management/commands/import_plos2EA.py

```diff
@@ -0,0 +1,495 @@
+from django.conf import settings as django_settings
+from django.core.management.base import BaseCommand
+import requests
+import datetime
+import json
+import urllib.request
+import os
+import django.utils.timezone
+from submission import models as submission_models
+from repository import models as repository_models
+from core import models as core_models
+from django.template.defaultfilters import slugify
+from uuid import uuid4
+from django.core.mail import send_mail
+
+import xml.etree.ElementTree as ET
+from ftplib import FTP_TLS
+from datetime import datetime
+import traceback
+import sys
+
+PRESS_ID = 1
+REPO_ID = 1
+COI_ID = 1
+DATA_ID = 1
+CCBY_ID = 1
+REPO_NAME = 'EarthArXiv (Dev)'
+SUCCESS_TO_MAIL = ['mahjabeen.yucekul@ucop.edu', 'alainna.wrigley@ucop.edu']
+FAILURE_TO_MAIL = ['mahjabeen.yucekul@ucop.edu', 'alainna.wrigley@ucop.edu']
+#SUCCESS_TO_MAIL = ['mahjabeen.yucekul@ucop.edu','eartharxiv_notifications@plos.org']

```

Comment:
I'd consider cleaning up commented config like this, unless it's necessary in the future.

Reply:
Done -  moved to settings.